src/app/inventory/nodes-table/models/generic.model.ts

```diff
@@ -10,8 +13,14 @@ export class GenericModel {
   isDeleted: boolean;
 
   constructor(params, type) {
+    params.custom_labels = {
+      mongokey: 'value1',
+      mongokey2: 'value2',
```

Comment:
what do these values mean?

Reply:
Oh  .. @darly-s  this was just an example from my  test case

Reply:
Yes, sorry, it need to be removed) It was code to test output